src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -0,0 +1,815 @@
+					break;
+				case ItemSelectionMode.Multiple:
+					newSelection = colView.SelectedItems;
+					break;
+			}
+
+			if (previousSelection.Count == newSelection.Count)
+			{
+				if (previousSelection.Count == 0 || (previousSelection[0] == newSelection[0]))
+				{
+					// Both selections are empty or have the same single item; no reason to signal a change
+					return;
+				}
+			}
+
+			var args = new SelectionChangedEventArgs(previousSelection, newSelection);
+			SelectionPropertyChanged(colView, args);
+		}
+
+        /// <summary>
+        /// Scrolling direction to display items layout.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public new Direction ScrollingDirection
+        {
+            get
+            {
+                return base.ScrollingDirection;
+            }
+            set
+            {
+                base.ScrollingDirection = value;
+
+                if (ScrollingDirection == Direction.Horizontal)
+                {
+                    ContentContainer.SizeWidth = ItemsLayouter.CalculateLayoutOrientationSize();
+                }
+                else
+                {
+                    ContentContainer.SizeHeight = ItemsLayouter.CalculateLayoutOrientationSize();
+                }
+            }
+        }
+
+        private void OnScrolling(object source, ScrollEventArgs args)
+        {
+            //ItemsLayouter.RequestLayout(ScrollingDirection == Direction.Horizontal ? args.Position.X : args.Position.Y);
+        }
+
+        /// <summary>
+        /// Adjust scrolling position by own scrolling rules.
+        /// Override this function when developer wants to change destination of flicking.(e.g. always snap to center of item)
+        /// </summary>
+        /// <param name="position">Scroll position which is calculated by ScrollableBase</param>
+        /// <returns>Adjusted scroll destination</returns>
+        /// <since_tizen> 8 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override float AdjustTargetPositionOfScrollAnimation(float position)
+        {
+            // Destination is depending on implementation of layout manager.
+            // Get destination from layout manager.
+            return ItemsLayouter.CalculateCandidateScrollPosition(position);
+        }
+
+        private View focusedView;
+        private int prevFocusedDataIndex = 0;
+
+        /// <inheritdoc/>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override View GetNextFocusableView(View currentFocusedView, View.FocusDirection direction, bool loopEnabled)
+        {
+            View nextFocusedView = null;
+
+            if (!focusedView)
```

Comment:
BaseHandle의 operator overloading에 의해 지원하는 방법이긴 하지만 Svace 등의 툴에서 감지하지 못해 Svace warning이 발생합니다. 따라서, NUI에서는 명시적으로 null과 비교하는 것을 권장하고 있습니다.
```suggestion
            if (focusedView == null)
```